SPECS/bmake/bmake.spec

```diff
@@ -48,6 +48,8 @@ sh ./make-bootstrap.sh
 
 %install
 export STRIP=/bin/true # Make sure binary is not stripped
+# skip 'job-output-null' which randomly fails in build pipelines
+export BROKEN_TESTS=job-output-null
```

Comment:
I think I've seen some logic inside the build files for `bmake`, where this test is already supposed to be skipped when the build runs in Bash (this should be the case here). Have you tried checking, why that doesn't trigger during our builds? Fixing that may be more future-proof.

Having said that, since 2.0 is getting deprecated, I think this is also good enough. I doubt we'll be updating this package.

Reply:
This test is automatically skipped for the `SCO_SV` OS but that's not triggered for AZL.